src/google/adk_community/memory/__init__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Community memory services for ADK."""
+
+from .open_memory_service import OpenMemoryService
```

Comment:
let's do lazy loading here. @wuliang229 to help provide some guidance on what's the proper way.

Reply:
let's do it in a separate PR.